### ISSUE TITLE
Feedback from beginner

### ISSUE THREAD

**Issue by nikeshashar** [makersacademy/preparetocode#25]
Prepare to code notes 

Terminal 
Where is the terminal
Add snippet to copy
Try and make it one page 

Homebrew
Make last button bigger
Update homebrew install image 
Make sure to type system password
4th step - typo (think / feeling ok) 
Type the small instructions 
And hit enter 

Ruby
Add quit instruction not close 
2nd step - rvm 
3rd step - typo 
Make button larger 

Heroku - 
Landing page - typos 
1st - 
2nd - link to open in a new window and fix image 

Guidance at the end to go to essential / useful screen 

Other 
Heroku keys add 
GitHub ssh keys 
Page title on tab

**Comment by natstar93** [comment]
How big do you want those buttons to be? And if the purpose of increasing the size is to make them stand out more how about making the background white and the text coloured also/instead?

**Comment by nikeshashar** [comment]
Are you referring to the final step buttons for Homebrew?

**Comment by natstar93** [comment]
Yep, and the 4th point under Ruby

**Comment by nikeshashar** [comment]
Yup - it's the next buttons not being very clear. I think we might need to rethink the colors but they are integrated into the platform

**Comment by nikeshashar** [comment]
But again that should be a different ticket. I would break out of this one and say make the next buttons larger

**Comment by nikeshashar** [comment]
I've turned this into a 5 pointer as it needs to be split out.
